### ISSUE TITLE
include-what-you-use.1: Non-standard layout

### ISSUE THREAD

**Issue by alejandro-colomar** [include-what-you-use/include-what-you-use#916]
Hi,

Your manual page is using a very non-standard layout.  To begin with, you should move the command (what is now after `USAGE:`) to a new section called `SYNOPSIS`.  Please read `man 7 man-pages` to get an overview of how a manual page should ideally be.

Thanks,

Alex

**Comment by alejandro-colomar** [comment]
Hmmmm.  It seems that it's a bug in Debian.  Not your fault.  I'm reporting it to them.  However, it's very weird: I don't know where they got the manual page from :/

**Comment by kimgr** [comment]
@alejandro-colomar I don't really know anything about manpages (and was hoping not to have to learn :)), the one in the IWYU tree was contributed by @aaronpuchert.

Maybe the Debian package predates our manpage? It was added on Nov 2, 2019 and included in IWYU 0.14.

**Comment by alejandro-colomar** [comment]
It's weird, because in their source repository, they have a page that looks like yours (I didn't diff it, but I think it's the same): <https://salsa.debian.org/pkg-llvm-team/iwyu/-/blob/master/include-what-you-use.1>.

But then, the files that end up in the package are very different: <https://packages.debian.org/bullseye/iwyu>.

I filed a bug to them: <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=988546>

**Comment by aaronpuchert** [comment]
> I don't know where they got the manual page from :/

Judging from

> To begin with, you should move the command (what is now after `USAGE:`) to a new section called `SYNOPSIS`.

it sounds like it was created from the `--help` output:

```
USAGE: include-what-you-use [-Xiwyu --iwyu_opt]... <clang opts> <source file>
Here are the <iwyu_opts> you can specify (e.g. -Xiwyu --verbose=3):
```

> Maybe the Debian package predates our manpage? It was added on Nov 2, 2019 and included in IWYU 0.14.

Sounds likely to me, the man page addition was relatively recent and maybe they "auto-generated" the man page before that.

**Comment by kimgr** [comment]
@aaronpuchert Thanks for the additional info!
